tool/genie.ts

```diff
@@ -109,7 +109,9 @@ const subcommands : { [key : string] : SubCommand } = {
     'upload-entity-values': require('./upload-entity-values'),
 
     'extract-translatable-annotations': require('./extract-translatable-annotations'),
-    'translate-schema-annotations': require('./translate-schema-annotations')
+    'translate-schema-annotations': require('./translate-schema-annotations'),
+
+    'synthetic-data-sampler': require('./synthetic-data-sampler')
```

Comment:
normally, we use a verb phrase for a command instead of using the class name directly here (both command name, and file name)